blocks/library/text/index.js

```diff
@@ -62,6 +77,7 @@ registerBlockType( 'core/text', {
 				} }
 				onMerge={ mergeBlocks }
 				style={ { textAlign: align } }
+				className={ `drop-cap-${ dropCap }` }
```

Comment:
Should this be a modifier class on Editable?

```js
className={ classnames( { 'has-drop-cap': dropCap } ) }
```

```css
.blocks-editable.has-drop-cap .blocks-editable__tinymce:not( :focus ) {
```

Or better yet, a prop on the component if it's Editable's own styles responsible for creating the effect?